ucsmsdk/ucssession.py

```diff
@@ -392,6 +392,29 @@ def _refresh(self, auto_relogin=False):
         self.__start_refresh_timer()
         return True
 
+    def __validate_ucsm(self):
+        """
+        Internal method to validate if connecting server is UCS.
+        """
+
+        is_ucs = False
+
+        from .ucsmethodfactory import config_resolve_class
+
+        nw_elem = config_resolve_class(cookie=self.__cookie,
+                                       in_filter=None,
+                                       class_id="networkElement")
+        try:
+            nw_elem_response = self.post_elem(nw_elem)
+            if nw_elem_response.error_code != 0:
+                self._logout()
```

Comment:
should this be `self._logout()` or `self._clear()`. The older code was doing a clear. Just want to double check